### PR TITLE
Add correct czech locales

### PR DESCRIPTION
**Pull request by martindybal**
There was a few mistake in czech translation and also czech translation was in weird folder so I moved it and I corrected it.

### CODE REVIEW

File: src/lang/cs.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    [
+      ['před %s vteřinami', 'za %s vteřiny'],
+      ['před %s vteřinami', 'za %s vteřin']
+    ],
+    [      ['před minutou', 'za minutu']    ],
+    [
+      ['před %s minutami', 'za %s minuty'],
+      ['před %s minutami', 'za %s minut']
+    ],
+    [      ['před hodinou', 'za hodinu']    ],
```
Comment (hustcc) [inline src/lang/cs.js]: please help me make the code style consistent. 😄

Comment (martindybal) [reply src/lang/cs.js]: Of course, what do you preffer? 

```
    [
      ['právě teď', 'právě teď']
    ],
    [
      ['před %s vteřinami', 'za %s vteřiny'],
      ['před %s vteřinami', 'za %s vteřin']
    ],
```

or
```
    [['právě teď', 'právě teď']],
    [
      ['před %s vteřinami', 'za %s vteřiny'],
      ['před %s vteřinami', 'za %s vteřin']
    ],
```

Comment (hustcc) [reply src/lang/cs.js]: all is ok, remove the redundant spaces will be ok.

Comment (martindybal) [reply src/lang/cs.js]: done

**Comment by martindybal** [comment]
@hustcc is there anything else?

**Comment by martindybal** [comment]
@hustcc Great, thank you. Can you also generate new version to npm?

**Comment by hustcc** [comment]
Can wait for this pr? https://github.com/hustcc/timeago.js/pull/171

**Comment by martindybal** [comment]
How long it will be take?

**Pull request merged**